code/modules/research/xenoarchaeology/machinery/coolant.dm

```diff
@@ -5,14 +5,13 @@
 	icon_state = "coolanttank"
 	amount_per_transfer_from_this = 10
 
-/obj/structure/reagent_dispensers/coolanttank/Initialize(mapload, ...)
+/obj/structure/reagent_dispensers/coolanttank/New()
```

Comment:
Initialize is more modern than New, iirc.

Reply:
The PR that edited coolant tanks on baystation12 replaced it with `New()`